make-it/EmMakefile

```diff
@@ -50,19 +62,33 @@ help:
 #  Vireo has grown as well, so emscripten is getting better if the over all size has shrunk by about 20%
 
 EMCC= emcc
-EM_DBGFLAG = -g4
-EM_OPTFLAG = -Os # -Os -O3
+
+# release config
+EM_OPTFLAG = -Os
+
+ifeq ($(BUILD), debug)
+   EM_OPTFLAG = -g4 -s ASSERTIONS=2 -s SAFE_HEAP=1
```

Comment:
I think with this change , custom.mak should be deleted. It should not have been checked in originally.